### PR TITLE
Inform users of app criteria

### PR DESCRIPTION
**Pull request by Hamster45105**
- Adds a GitHub action to automatically respond to new app requests with a link to the app criteria file.
- Adds a link to the app criteria on the modal after clicking the request apps button.

### CODE REVIEW

File: data/strings.json [hunk 1]
```diff
@@ -42,17 +42,11 @@
...
-    },
-    "modalBody2": {
-        "en": "At a minimum, when making your request, you must provide a link to the app download page and any configs/things you have tried to get the app working so far. Please have a go at adding it yourself before requesting.",
-        "de": "Wenn du eine Anfrage stellst, musst du zumindest einen Link zur Download-Seite der App angeben und alle Konfigurationen/Dinge, die du bisher ausprobiert hast, damit die App funktioniert mitteilen. Bitte versuche, die App selbst hinzuzufügen, bevor du eine Anfrage stellst."
+    "appCriteria": {
+        "en": "app criteria"
     },
     "modalAcceptButton": {
-        "en": "I have read understand the above information and understand that if I have not followed the above insturctions, my request may be ignored.",
-        "de": "Ich habe die obigen Informationen gelesen und verstanden, dass mein Antrag ignoriert wird, wenn ich die obigen Hinweise nicht beachtet habe."
```
Comment (ImranR98) [inline data/strings.json:55]: Should the `de` prop be removed?

Comment (DwainZwerg) [reply data/strings.json:55]: It doesn't get any shorter than this in German: „Ich habe die obigen Informationen gelesen und verstehe, dass mein Antrag ignoriert werden wird, wenn ich die Informationen nicht beachte.“

Comment (DwainZwerg) [reply data/strings.json:55]: Could it be that the German text for the modalAcceptButton1 is missing? That would be "Ich habe die"

Comment (DwainZwerg) [reply data/strings.json:55]: Ah, I've seen something else (which I hadn't told you before): modalButtonText schould have "Weiter".

**Comment by DwainZwerg** [comment]
@Hamster45105 I hadn't seen that it already said "Fortfahren". That's even more appropriate. Either way, some kind of json error was built into the latest commit.

**Pull request merged**